### PR TITLE
Revised 'Getting Started' for clarity

### PR DESCRIPTION
**Pull request by xjohn600**
Revised sentences in 'Getting Started' category for clarity
Fixed grammar mistakes

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -12,11 +12,9 @@ Airtest
 Getting Started
 ---------------
 
-Airtest is a cross-platform automated testing framework with main focus on games,
-which can also be used for native apps. Currently, Windows and Android are well supported.
-Support for iOS comes in near future.
+Airtest is a cross-platform automated testing framework focused mainly on games, but can also be used for native apps. Windows and Android are currently supported. iOS support will be coming in the near future.
 
-Airtest provides cross-platform APIs, including app installation, simulated input, assertion and so forth. Airtest uses image recognition technology to locate UI elements, so that you can automate test on games without injecting any code. After running the test, an HTML report will be generated automatically, that allows you to quickly locate failed test points.
+Airtest provides cross-platform APIs, including app installation, simulated input, assertion and so forth. Airtest uses image recognition technology to locate UI elements, so that you can automate test on games without injecting any code. The test will generate a HTML report, which allows you to quickly locate failed test points.
```
Comment (Meteorix) [inline README.rst]: I think it should be "an HTML report"?

**Comment by xjohn600** [comment]
Yeah it is. Sorry, fixed it

**Comment by Meteorix** [comment]
@xjohn600 good revision, thanks! 

I left you a review comment, please take a look. And I have updated README, so maybe you need to merge it and pr again.

**Comment by xjohn600** [comment]
Okay, thanks I merged and requested a new pr. Thanks for considering my pr!

**Pull request closed without merging**